.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You'll need to rebase this branch and ignore `.vscode`. You should set up a [global gitignore file](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)!

Reply:
Will do.